test/integration2/dialects/postgresql.spec.js

```diff
@@ -0,0 +1,119 @@
+const { expect } = require('chai');
+const {
+  getAllDbs,
+  getKnexForDb,
+  Db,
+  getDbTestConfig,
+} = require('../util/knex-instance-provider');
+
+describe('PostgreSQL dialect', () => {
```

Comment:
does this not work with Cockroach?

Reply:
no idea, i have never used Cockroach and don't know much about it

Reply:
can you try ensbling tests for Cockroach too and see if it passes?
crdb extends pg dialect